pkg/auth/signing/encode.go

```diff
@@ -0,0 +1,28 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package signing
+
+import (
+	"github.com/gogo/protobuf/proto"
+
+	"storj.io/storj/pkg/pb"
+)
+
+func EncodeOrderLimit(limit *pb.OrderLimit2) ([]byte, error) {
+	signature := limit.SatelliteSignature
```

Comment:
Shouldn't this have also `limit.SatelliteSignature = nil`?

Reply:
yes